directory_size_manager:
  is_create_directory_if_doesnt_exist: true
  max_size_in_kilobytes: 1 #1048576
